plugins/packages/packages_inspector.rb

```diff
@@ -90,12 +90,22 @@ def inspect_dpkg
       )
     end
     packages.each_slice(100) do |packages_slice|
-      apt_cache_output = @system.run_command(
-        "apt-cache",
-        "show",
-        *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
-        stdout: :capture
-      )
+      begin
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
+          stdout: :capture
+        )
+      rescue
```

Comment:
Using exceptions for things like that is quite a hack. Would it be possible to use the version of the command or so to know which format to use beforehand?